doc/dox_comments/header_files/ssl.h

```diff
@@ -680,6 +701,25 @@ WOLFSSL_API int wolfSSL_dtls_set_export(WOLFSSL* ssl, wc_dtls_export func);
 WOLFSSL_API int wolfSSL_dtls_export(WOLFSSL* ssl, unsigned char* buf,
                                                               unsigned int* sz);
 
+/*!
+    \brief Used to export a serialized TLS session. In most cases
+    wolfSSL_set_session should be used instead, this function is for importing a
```

Comment:
This section also has some confusing language about using `wolfSSL_set_session` and importing. Can this be improved?